main.cpp

```diff
@@ -364,9 +364,9 @@ void Racecar::accelerate(unsigned int currSpeed)
     }
 }
 
-int Racecar::currGear()
+int Racecar::currentGear(int currGear)
```

Comment:
`currentGear` is a property, not an action.
Your function name needs a verb in it. 
`getCurrentGear()` would be appropriate, **assuming** your class has a `currentGear` member variable that stores the `Racecar`'s current gear.
You could also add a `setCurrentGear(int gearNumber)` function that sets the value of this `currentGear` member variable.
- [ ] revise